src/core/jit/builder.rs

```diff
@@ -51,6 +54,7 @@ pub struct Builder {
     pub document: ExecutableDocument,
 }
 
+// TODO: make generic over Value (Input) type
```

Comment:
Not sure if we can make this happen because we depend on async_graphql for parsing the request and just abstracting over the input type won't be enough.